README.md

```diff
@@ -36,3 +36,10 @@ There are two flags which need to be set:
 ## Related links
 * Running demo application - [kbss/fta-fmea-demo](https://kbss.felk.cvut.cz/fta-fmea-demo/) 
 * Backend project - [fta-fmea](https://github.com/kbss-cvut/fta-fmea)
+
+## Dockerization
+1. Download [GraphDB](https://graphdb.ontotext.com/) standalone server ZIP archive and place it into the `deploy/db-server` folder.
```

Comment:
Just curious if we can have a shell script called 
'./deploy/db-server/bin/download-graphdb.sh` script to do that from environment settings ?

Reply:
Yes, we can, ill do that

Reply:
We can't download it directly, first we need to fill form and then download it from email.
And I'm not sure how long this link will be available.

The link from email looks like : 
https://email.ontotext.com/e3t/Ctc/GD+113/cGJhF04/VWcV8x3WHrJ3W8mhwjv35SMRCW8cYYgJ4L1WwfN39GZDG3q3pBV1-WJV7CgFvbW1WVgFZ4YHJL6N3g9W85tCDKRN2VXCjph3CTSVf-kjf8vBTNKW6MdSGT8-7vy0W8bsPBX84NDWJW3jJcyL7KcTYFW1ghc-p5Ttt8zW1fvmTN1lNjscW8ywD756g74N9W3pQ8GG6-JQwYW1H5cwQ6FvvKkW35KyNK69t1kjW974Bjy98rB8HW62KhtP1wZtT-W5f6d2B5t_6GcW6kp3DR1z-fjXW18xKrH8LPS27W2q3NcY5WrQnDW2D_n4Y4dPVWDW2YzlbK8hJYBbW4wYkMj6BDGzkW5xGyL57vq8-dN6gBw_ppXD9YW36jWGX8XNjbVW5GGBHx11m8KqW8NqzzL7p3YLfW7xmPNp4YsZWrW186bFM9b12ZGW3-4-Ks1tgcQ83cBj1

@blcham